projects/fal/src/fal/_serialization.py

```diff
@@ -97,6 +97,31 @@ def pickle_private_attr(field: ModelPrivateAttr) -> tuple[Callable, tuple]:
     _register(ModelPrivateAttr, pickle_private_attr)
 
 
+def _patch_pydantic_model_serialization() -> None:
+    # If user has created new pydantic models in his namespace, we will try to pickle those
+    # by value, which means recreating class skeleton, which will stumble upon
+    # __pydantic_parent_namespace__ in its __dict__ and it may contain modules that happened
+    # to be imported in the namespace but are not actually used, resulting in pickling errors.
+    # Unfortunately this also means that `model_rebuid()` might not work.
```

Comment:
Doesn't seem like we were using v1 counterpart `update_forward_refs` either, so shouldn't be a big deal.

Trying to also think how to properly fix it in pydantic so that we don't have to keep patching it...